.github/workflows/ci.yaml

```
@@ -0,0 +1,126 @@
+    with:
+      branch: ${{ needs.changes.outputs.branch }}
+
+  always-run:
+    # used in branch protection rulesets as the only relevant step needing to pass
+    name: CI Status
+    runs-on: ubuntu-latest
+    needs: [lint, tests, android, desktop, docker]
+    if: always()
+    steps:
+      - name: Fail if any dependent job failed
+        run: |
```

actionlint (1.7.8)

got unexpected character '$' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)